maps/kondaru.dmm

```diff
@@ -58278,8 +58278,7 @@
 "wFB" = (
 /obj/storage/cart/mechcart{
```

Comment:
this should use the `/obj/storage/cart/mechcart/breach` subtype instead, probably?